test/09-contracts.ts

```diff
@@ -65,6 +65,34 @@ describe('contracts', () => {
 			).to.not.be.undefined;
 		});
 
+		it('should normalize the assets included with the contracts', async () => {
+			mockRepo('balena-io', 'contracts', 'base-contracts');
+			mockRepo('balena-io', 'other-contracts');
+			await fetchContractsLocally([
+				{ owner: 'balena-io', name: 'contracts' },
+				{ owner: 'balena-io', name: 'other-contracts' },
+			]);
+
+			const contracts = await getContracts('hw.device-type');
+			const rpi3Contract = contracts.find(
+				(contract) => contract.slug === 'raspberrypi3',
+			);
+			const otherDtContract = contracts.find(
+				(contract) => contract.slug === 'other-contract-dt',
+			);
+
+			expect(rpi3Contract?.assets?.logo).to.exist;
```

Comment:
NIT: Do we really need all those `?.` (and also repeat in every line)?
I mean we just mocked that.
If we still want to, then it might make sense to `expect().to.have.nested,property('assets.logo')`.

Reply:
Typescript wants them, and I honestly prefer those over checking if it has a nested property.